pkg/cassandra/config/config.go

```diff
@@ -58,6 +58,18 @@ type Schema struct {
 	// while connecting to the Cassandra Cluster. This is useful for connecting to clusters, like Azure Cosmos DB,
 	// that do not support SnappyCompression.
 	DisableCompression bool `mapstructure:"disable_compression"`
+	// Datacenter is the name for network topology
```

Comment:
we need to add `CreateSchema bool` (default false) for backwards compatibility

Reply:
You mean `CreateSchema` as a config key, based on which we will create schema? Makes sense, will add it.

Reply:
Hi @yurishkuro , I was thinking would `CreateSchema` key logically fit in `Schema` or should we declare it outside of `Schema`? Any views on it? Thanks

Reply:
```
cassandra:
  schema:
    create: true
```

Reply:
Added